.github/workflows/ax.yml

```diff
@@ -68,14 +68,17 @@ jobs:
         # Only test unified builds (core+ax). weekly CI should test standalone
         # @note  LLVM 17 builds are incompatible with the vdb python plugin on linux
         config:
+          - { image: '2026',           cxx: 'clang++', build: 'Release', cmake: '' }
```

Comment:
Do we want to use a specific clang version here? `2026-clang20.0` ?

This is now a lot of AX builds, is it possible that we could move a few of the in-between versions to the weekly build? If it builds on the earliest and latest supported, we should have reasonable confidence that it will build on the versions in between.